### ISSUE TITLE
Role fails when attempting to create systemd server properties

### ISSUE THREAD

**Issue by mok0** [nolte/ansible-minecraft#43]
I am running Ubuntu 19.04, when running a simple playbook calling the nolte.minecraft role I get an error telling me a password is required. I don't understand this, since I have "become: true" in the playbook:

````
TASK [nolte.minecraft : generate server.properties] ****************************
fatal: [localhost]: FAILED! => {"changed": false, "module_stderr": "sudo: a password is required\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}
````

When running the playbook again using `sudo` the playbook fails again, this time with another error message:

```
 TASK [nolte.minecraft : update server.properties] ******************************
fatal: [localhost]: FAILED! => {"changed": false, "module_stderr": "<stdin>:17: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses\nTraceback (most recent call last):\n  File \"<stdin>\", line 113, in <module>\n  File \"<stdin>\", line 105, in _ansiballz_main\n  File \"<stdin>\", line 48, in invoke_module\n  File \"/usr/lib/python3.7/imp.py\", line 234, in load_module\n    return load_source(name, filename, file)\n  File \"/usr/lib/python3.7/imp.py\", line 169, in load_source\n    module = _exec(spec, sys.modules[name])\n  File \"<frozen importlib._bootstrap>\", line 630, in _exec\n  File \"<frozen importlib._bootstrap_external>\", line 728, in exec_module\n  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n  File \"/tmp/ansible_minecraft_server_file_payload_8skfdwn5/__main__.py\", line 375, in <module>\n  File \"/tmp/ansible_minecraft_server_file_payload_8skfdwn5/__main__.py\", line 345, in main\n  File \"/tmp/ansible_minecraft_server_file_payload_8skfdwn5/__main__.py\", line 152, in __init__\nAttributeError: 'dict' object has no attribute 'iteritems'\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}

```

I'm not sure whether this error message comes from Ansible or from the minecraft role, however, end result is that no files are produced for `systemd`.

**Comment by nolte** [comment]
Hi, thanks for the Feedback, 19.04 was never tested on my side, i think the Problem is the Python Version, and the custom Lib [minecraft_server_file.py](https://github.com/nolte/ansible-minecraft/blob/master/library/minecraft_server_file.py). 

I will try to find a way to Reproduce it. Can you give me the full log with --vvv for finding the Problem?

**Comment by frank-u** [comment]
Ubuntu 19.04 and newest doesn't contain python2, so - Ansible use python3 on the target machines. Current code is not compatible to run on it. PR #54 is addressed to fix that.

**Comment by nolte** [comment]
Thanks for fixing!

The bugfix is published by Release [v5.0.10](https://github.com/nolte/ansible-minecraft/releases/tag/v5.0.10)
